test('My first Test', () => {
    throw new Error('My first error');
});